cmd/flow_capture.go

```diff
@@ -147,16 +147,20 @@ func runFlowCaptureOnAddr(port int, filename string) {
 		// terminate capture if max bytes reached
 		totalBytes = totalBytes + int64(bytes)
 		if totalBytes > maxBytes {
-			log.Infof("Capture reached %s, exiting now...", sizestr.ToString(maxBytes))
-			return
+			if exit := onLimitReached(); exit {
+				log.Infof("Capture reached %s, exiting now...", sizestr.ToString(maxBytes))
+				return
+			}
 		}
 
 		// terminate capture if max time reached
 		now := currentTime()
 		duration := now.Sub(startupTime)
 		if int(duration) > int(maxTime) {
-			log.Infof("Capture reached %s, exiting now...", maxTime)
-			return
+			if exit := onLimitReached(); exit {
+				log.Infof("Capture reached %s, exiting now...", sizestr.ToString(maxBytes))
```

Comment:
(I said it already in another PR)
this should be maxTime here

Reply:
thanks ! https://github.com/netobserv/network-observability-cli/pull/102/commits/71c0bcb7765ef48c5eac6edcaa10690251387db0